task1/src/test/java/com/fundacionjala/movies/TestChildrenMovie.java

```diff
@@ -0,0 +1,24 @@
+package com.fundacionjala.movies;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+/**
+ * Created by Mijhail on 14/06/2016.
+ */
+public class TestChildrenMovie {
+
+    @Test
+    public void calculateChargeMovieFor1DayRental() {
+        Movie movie = new ChildrenMovie("X-Men");
```

Comment:
If you notice you are repeating the Movie instance in all the tests
Please add a Before method
```java
private Movie movie;

@Before
public void setUp() {
    movie = new ChildrenMovie("X-Men");
}
```